.gitignore

```diff
@@ -1,2 +1,3 @@
 /.metadata/
 build/
+.vscode/
```

Comment:
This just  added is to be ignored, files are still present in the version control.

Reply:
language

Reply:
HAPPY?

Reply:
yes